examples/bioc/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+name = "bioc"
+version = "3.0.1"
+authors = ["Parity Technologies <admin@parity.io>"]
+edition = "2021"
+publish = false
+
+[dependencies]
+ink_primitives = { version = "3.0", default-features = false }
+ink_metadata = { version = "3.0", default-features = false, features = ["derive"], optional = true }
+ink_env = { version = "3.0", default-features = false }
+ink_storage = { version = "3.0", default-features = false }
+ink_lang = { version = "3.0", default-features = false }
+ink_prelude = { version = "3.0", default-features = false }
```

Comment:
Can you please check the manifest of the other examples. You should use a `path` dependency here. This is probably the reason why the waterfall fails.